Enhanced PDF and Graphical Report Design for Improved Clarity and Presentation .

EQC16 opened the pull request:
### 📝 **Description:**  
This update refines the layout, styling, and visual elements across multiple report templates to enhance readability, professionalism, and consistency in generated PDFs and graphical reports.  

#### **Main Changes:**  
- **`advancePaymentGraphReport.blade.php`**  
  - Added clear labels above each chart (Bar, Line, Pie, and Donut) for better visual identification.  
  - Improved spacing and alignment for a more professional layout.  

- **`generateIncidentListReport.blade.php`**  
  - Introduced a new `.oval_color` badge style to visually represent incident statuses with corresponding colors.  
  - Replaced plain status text with color-coded labels for better readability.  
  - Removed the “Descripción” column to streamline the table’s appearance.  

- **`generateIncidentStatusListReport.blade.php`**  
  - Applied the same `.oval_color` badge styling to the incident status table.  
  - Status names are now displayed in uppercase within a colored badge reflecting their state.  

- **`pdfCustomers.blade.php`**  
  - Optimized page setup with `@page` definition and corrected background image orientation.  
  - Adjusted logo size and layout for a more balanced header.  
  - Centered and emphasized the committee title section.  
  - Simplified customer address representation into a single “Dirección” column.  

- **`pdfCustomersSummary.blade.php`**  
  - Reduced font size, line height, and margins for improved space usage and visual density.  
  - Adjusted connection detail formatting for a cleaner presentation.  

#### **Benefits:**  
These visual and structural improvements make reports more organized, easier to read, and visually consistent across all PDF and chart-based documents.

EQC16 committed:
Enhanced PDF and Graphical Report Design for Improved Clarity and Presentation.

And-Y21 commented on resources/views/reports/pdfCustomers.blade.php:
```diff
@@ -33,17 +38,16 @@
-                padding-top: 60px;
                 font-family: 'Montserrat', sans-serif;
-                font-size: 18pt;
+                font-size: 20pt;
```
Lo mismo para esta parte, es información que esta en todos los reportes.

And-Y21 commented on resources/views/reports/pdfCustomers.blade.php:
```diff
@@ -128,14 +132,13 @@
-                width: 100px;
-                height: 100px;
+                width: 120px;
+                height: 120px;
```
Lo mismo aqui

And-Y21 commented on resources/views/reports/pdfCustomers.blade.php:
```diff
@@ -159,15 +162,13 @@
-                                COMITÉ DEL SISTEMA DE AGUA POTABLE DE {{ $authUser->locality->name }}, {{ $authUser->locality->municipality }}, {{ $authUser->locality->state }}
-                            </p>
-                        </div>
-                    </td>
```
¿Por qué borrarlo y volverlo a poner?

EQC16 replied:
Por que al momento de ajustarse para que este en el formato vertical del pdf estaba dentro de una tabla con el logo y no podias ajustar bien el titulo enotnces saque el titulo de la tabla para que se pueda manipular y ajustar bien y no quede dentro o a lado del logo si no justo abajo.

EQC16 committed:
change in the incident view

The use of some incorrectly placed variables was changed for a better, more uniform view.
